webhook_receiver_shopify/views.py

```diff
@@ -130,9 +131,11 @@ def order_update(_, conf, data):
         logger.info('Created order %s' % order.order_id)
     else:
         logger.info('Retrieved order %s' % order.order_id)
+    
+    if order.status == Order.NEW and ENROLL_TAG in payload['tags']:
```

Comment:
@ngdangdat So we noticed during earlier testing that an order update does not have a "NEW" status. So could you remove that condition and only keep the condition that checks the `ENROLL_TAG`?

Reply:
Hi @pkulkark, thanks for the information. I updated the source code. Kindly check! Thanks!

One question on this, is there a case that "GIFTCARD" and "AUTHORIZED" exist in a same `payload['tags']` of a request from webhook? In that case, how should we handle the logic here?